modules/custom/d_documentation/pages/d_p_tiles.yml

```diff
@@ -185,13 +188,16 @@ field_page_section:
               alt: 'Title / Subtitle'
             -
               path: d_demo/d_demo_tiles_2_3.jpg
-              alt: '2x2 Title / Subtitle'
+              alt: 'Title / Subtitle'
             -
               path: d_demo/d_demo_tiles_2_4.jpg
               alt: 'Title / Subtitle'
             -
               path: d_demo/d_demo_tiles_2_5.jpg
               alt: 'Title / Subtitle'
+        field_d_settings:
+          type: field_p_configuration_storage
+          featured_images: 1
```

Comment:
w sumie to tutaj te `featured_images` chyba powinien być ustawiony na 3 :thinking: